RInAppMessaging/Classes/Views/FullView.swift

```diff
@@ -109,6 +110,17 @@ internal class FullView: UIView, FullViewType, RichContentBrowsable {
         bodyViewOffsetYConstraint.constant = hasImage ? 0 : uiConstants.bodyViewSafeAreaOffsetY
     }
 
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        let exitButtonCenter = convert(exitButton.center, from: exitButton.superview)
```

Comment:
Do we have any way to test this code?
Maybe we need UI tests (doesn't need to be in this PR)

Reply:
Yeah. I think it's time to write some UI tests